Cooper/ClientApp/src.v2/app/layouts/profile/profile.layout.ts

```diff
@@ -95,6 +96,8 @@ export class ProfileLayoutComponent implements OnInit, OnDestroy {
     }
 
     this.isOwnProfile = this.sessionService.GetSessionUserId() === this.profile.id;
+
+    this.isSubscription = this.isOwnProfile ? true : this.sessionService.isSubscription(this.profile.id);
```

Comment:
this.isSubscription = this.isOwnProfile || this.sessionService.isSubscription(this.profile.id);